src/ui/components/Collections/CollectionRow.jsx

```diff
@@ -27,11 +28,13 @@ function CollectionRow(props) {
     const bundle = {};
     headers.forEach(header => {
       if (!header.viewOnly) {
-        bundle[header.value] = state[header.value];
+        const value = state[header.value];
+        if (value) bundle[header.value] = value;
       }
     });
 
-    axios.put(`/collection/${selectedCollection}?id=${bundle.id}`, bundle).then(() => {
+    const query = bundle.id ? `id=${bundle.id}` : `fullUrl=${base64url(bundle.fullUrl)}`;
```

Comment:
I'm not clear on why this changed. Are there cases where we don't have an id?

Reply:
I don't think it was correct before. IDs are not unique for our FHIR collections. The backend crud handler supported fullUrl query params but the UI wasn't using them